### ISSUE TITLE
Exclude certain File Types in the index searching results.

### ISSUE THREAD

**Issue by ComputerCulture** [Flow-Launcher/Flow.Launcher#2144]
Hello,
It would be great to have somewhere in the settings, to have a field box you can add exclude/Ignore certain file types. 
in the search results from not being indexed or showing up in the results.

Examples are (.jpg .mov .mp3 .docx) and other types a user can add.

Thank you for our work, and this wonderful software.

**Comment by deefrawley** [comment]
Hi. You can exclude file types from either the Windows or Everything search depending on which one you use. You need to add these to each search type's settings. Is this what you were after or did you mean on a per search basis?

**Comment by ComputerCulture** [comment]
Hi, thank you for your reply. What I was looking for if possible (see screenshot image)
![FluentSearch - Ignored file extentions](https://github.com/Flow-Launcher/Flow.Launcher/assets/133943765/6cc1feb5-f5df-49b3-8196-1c88d2106dc8)
There is another program called "FluentSearch" and in the settings you can Ignore certain file types from search results.
This is what I was looking for, if this feature can be added to the settings in Flow Launcher.
Thank you in advance.

**Comment by taooceros** [comment]
I think we can add an additional filter to this.

**Comment by ComputerCulture** [comment]
> I think we can add an additional filter for this.

Wonderful, that would be a great feature. 
Thank you, looking forward to the addition
All the best...

**Comment by ComputerCulture** [comment]
Hello,
Any news if this "add an additional filter" can be added to the software.
Thank you...

**Comment by gunrunner20** [comment]
I can just say I second this feature request, but additionally it would help me a lot if also universal folder paths could be excluded, like "*/archive".

Currently just specific paths (without wildcard) are supported to be excluded.

**Comment by EnigmaSolved** [comment]
> I can just say I second this feature request, but additionally it would help me a lot if also universal folder paths could be excluded, like "*/archive".
> 
> Currently just specific paths (without wildcard) are supported to be excluded.

I agree. I use Everything to index my NAS and I would like to exclude the `#recycle` folders for all shares. Within Everything I have a filter to exclude these by default, and then another filter for when I want to specifically search `#recycle`.
